### PR TITLE
Add info abt node material in morph targets docs

### PR DESCRIPTION
**Pull request by carolhmj**

### CODE REVIEW

**Review by Popov72 — APPROVED** [review]
Great!

I guess the same thing could be done with the **Instances** (to support instances) and **Bones** (to support rigged characters) blocks. Maybe we should create an issue for that?

**Review by RaananW — APPROVED** [review]
@carolhmj - waiting for your answer to the last comment, you can merge right after :-)

**Comment by sebavan** [comment]
@PirateJC can you force a rebuild ? I can not seem to have access :-(

**Comment by sebavan** [comment]
@RaananW could you have a look at this one ? I can not understand why the build is failing :-(

File: content/features/featuresDeepDive/mesh/morphTargets.md
Comment (RaananW) [inline content/features/featuresDeepDive/mesh/morphTargets.md]:
```suggestion
<Playground id="#HPV2TZ#299" title="Using Node Material with Morph Targets" description="Use Morph Target node on the Node Material"/>
```

**Comment by RaananW** [comment]
> @RaananW could you have a look at this one ? I can not understand why the build is failing :-(

Should be good to go now

**Comment by sebavan** [comment]
@carolhmj you might need to merge master back in :-)

**Comment by carolhmj** [comment]
> Great!
> 
> I guess the same thing could be done with the **Instances** (to support instances) and **Bones** (to support rigged characters) blocks. Maybe we should create an issue for that?

@Popov72 great pointed, added these too!

**Pull request merged**
